packages/server-wallet/src/protocols/open-channel.ts

```diff
@@ -191,10 +174,9 @@ const signPreFundSetup = (ps: ProtocolState): ProtocolResult | false =>
   });
 
 const signPostFundSetup = (ps: ProtocolState): ProtocolResult | false =>
-  myTurnToPostfund(ps) &&
-  (isFunded(ps) || isFakeFunded(ps.app)) &&
   ps.app.latestSignedByMe &&
-  ps.app.supported &&
+  ps.app.latestSignedByMe.turnNum < ps.app.participants.length &&
```

Comment:
Should we check that there exists a supported prefund state before signing a postfund state?